arkdep-build.d/test-manjaro-gnome/overlay/post_bootstrap/etc/dconf/db/local.d/01-manjaro-theming

```diff
@@ -7,9 +7,10 @@ primary-color='#3465a4'
 secondary-color='#000000'
 
 [org/gnome/desktop/interface]
+accent-color='maia'
```

Comment:
When this works also with GNOME 46 and get ignored, fine with me.

Reply:
I validated the DB still builds fine under GNOME 46.